teeracle/src/tests.rs

```diff
@@ -25,12 +25,18 @@ use test_utils::ias::consts::{
 	TEST5_SIGNER_PUB, TEST8_MRENCLAVE, URL,
 };
 
+const COINGECKO_SRC: &[u8] = "https://api.coingecko.com".as_bytes();
+const COINMARKETCAP_SRC: &[u8] = "https://coinmarketcap.com/".as_bytes();
+
+const DOT_USD_TRADING_PAIR: &[u8] = "DOT/USD".as_bytes();
+const TEER_USD_TRADING_PAIR: &[u8] = "TEER/USD".as_bytes();
+
```

Comment:
You can save quite some `to_owned`s below if you declare those as `Vec<u8>`

Reply:
There are constants, I don't know how I can declare them as Vec<u8>

Reply:
Like this: `const COINGECKO_SRC: Vec<u8> = "https://api.coingecko.com".as_bytes().to_vec();`

Reply:
btw. `to_owned` is the same as `to_vec` for `&[u8]`

Reply:
to_owned or to_vec throw an error for constant:
 `calls in constants are limited to constant functions, tuple structs and tuple variants`.
 as const it can be &[u8] or &str, but not Vec or String

Reply:
Ah ok I see. Makes sense. But const can be vecs. The problem is that `to_vec` is not a `const fn`. But there is no way to create a non-empty vec with `const fn`s as it seems. :)